ui/component/notification/index.js

```diff
@@ -1,8 +1,14 @@
 import { connect } from 'react-redux';
```

Comment:
When I press the Creator Like (Heart), a medium-sized avatar gets stuck in the screen.

![image](https://user-images.githubusercontent.com/64950861/129561687-22626129-7fb8-49ad-a705-ba4805a9ff4f.png)

Reply:
Noticed this too, probably minor but should be fixed.